network/network_test.go

```diff
@@ -257,4 +257,51 @@ var _ = Describe("Test Network", func() {
 			})
 		})
 	})
+
+	Describe("Test GetNumEndpointsInNetNs", func() {
+		Context("When network exists", func() {
+			It("Should return two endpoints", func() {
+				netNs := "989c079b-45a6-485f-8f9e-88b05d6c55c5"
+				networkID := "byovnetbridge-vlan1-10-128-8-0_23"
+				nm := &networkManager{
+					ExternalInterfaces: map[string]*externalInterface{
+						networkID: {
+							Name: networkID,
+							Networks: map[string]*network{
+								"byovnetbridge-vlan1-10-128-8-0_23": {
```

Comment:
to closely mimic the existing usecase, can you add one endpoint in each network instead of 2 endpoints in single network? so need 2 networks, each having one endpoint with given netNs

Reply:
this test case is kept and new test case (two networks + two endpoints) is added